lib/livebook_web/components/app_components.ex

```diff
@@ -133,62 +131,25 @@ defmodule LivebookWeb.AppComponents do
       </div>
     </div>
 
-    <%= if Hubs.Provider.type(@hub) == "team" do %>
-      <div class="flex flex-col">
-        <div class="grid grid-cols-1 md:grid-cols-2 gap-3">
-          <.select_field
-            label="Zero Trust Authentication provider"
-            field={@form[:zta_provider]}
-            help={
-              ~S'''
-              Enable this option to generate
-              Livebook Dockerfiles with proxy
-              authentication for deployed
-              notebooks
-              '''
-            }
-            prompt="None"
-            options={zta_options()}
-            disabled={@disabled}
-          />
-
-          <.text_field
-            :if={zta_metadata = zta_metadata(@form[:zta_provider].value)}
-            field={@form[:zta_key]}
-            type={Map.get(zta_metadata, :input, "text")}
-            label={zta_metadata.value}
-            placeholder={Map.get(zta_metadata, :placeholder, "")}
-            phx-debounce
-            disabled={@disabled}
-          />
-        </div>
-
-        <div :if={zta_metadata = zta_metadata(@form[:zta_provider].value)} class="text-sm mt-1">
-          See the
-          <a
-            class="text-blue-800 hover:text-blue-600"
-            href={"https://hexdocs.pm/livebook/#{zta_metadata.type}.html"}
-          >
-            Authentication with <%= zta_metadata.name %> docs
-          </a>
-          for more information.
-        </div>
-      </div>
-    <% end %>
+    <div class="flex flex-col gap-2">
+      <.checkbox_field
+        field={@form[:zta_provider]}
+        label="Enable Livebook Teams authentication"
```

Comment:
```suggestion
        label="Enable authentication via Livebook Teams"
```

cc @hugobarauna

Reply:
Or perhaps:

```suggestion
        label="Authenticate via Livebook Teams"
```

Since it is the label of a checkbox anyway.